# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/kasc/libsdl_perl/console_fix.patch
# ROCK Linux is Copyright (C) 1998 - 2003 Clifford Wolf
# 
# This program is free software; you can redistribute it and/or modify
# it under the terms of the GNU General Public License as published by
# the Free Software Foundation; either version 2 of the License, or
# (at your option) any later version. A copy of the GNU General Public
# License can be found at Documentation/COPYING.
# 
# Many people helped and are helping developing ROCK Linux. Please
# have a look at http://www.rocklinux.org/ and the Documentation/TEAM
# file for details.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

--- ./SDL_perl.xs	2003-01-03 06:14:35.000000000 +0100
+++ ./SDL_perl.xs	2003-05-26 16:04:15.000000000 +0200
@@ -67,7 +67,7 @@
 
 #ifdef HAVE_SDL_CONSOLE
 #include "CON_console.h"
-#include "CON_consolecommands.h"
+/*#include "CON_consolecommands.h"*/
 #include "DT_drawtext.h"
 #endif
 
